ksql-engine/src/main/java/io/confluent/ksql/planner/plan/AggregateNode.java

```diff
@@ -203,17 +205,18 @@ public WindowExpression getWindowExpression() {
 
     final Serde<GenericRow> genericRowSerde = builder.buildGenericRowSerde(
         valueSerdeFactory,
-        aggregateArgExpanded.getSchema().getSchema(),
+        PhysicalSchema.from(aggregateArgExpanded.getSchema(), SerdeOption.none()),
```

Comment:
The serde options the user supplies affect the source and sink topics only.  At the moment the only serde options are the ones supplied by the user. Hence internal topics take `SerdeOption.none()`.  Personally, I think that's pretty clean and self explanatory.

However, in the future there may well be options that do make sense on internal topics.  For example, the AggregateNode currently includes the ROWKEY and ROWTIME in the serialized value.  One thought I've had for clearing up the whole with/without implicits thing is to handle this on the serde side using a `SerdeOption`.  So in the future we may see `SerdeOption.of(SerdeOption.INCLUDE_ROWTIME)` here. TBD

I'm not keen on adding comments to code like:

```java
// SerdeOption of none because its an internal topic.
PhysicalSchema.from(aggregateArgExpanded.getSchema(), SerdeOption.none())
```

As I see them adding little benefit, and they end up out of date and cause more issue than they solve _most_ of the time IMHO. 

Thoughts?